Optimizing the Security of your Skywallet.md

```diff
@@ -0,0 +1,48 @@
+# Optimizing the Security of your Skywallet
+
+Even though the Skywallet provides greater security when compared to a Desktop Wallet, it is essential to follow these security optimizations to get maximum security benefit from your Skywallet.
+
+Two things are necessary for a third party to access your funds in Skywallet, which is either the PIN code of the Skywallet or the Seed of the Skywallet.
```

Comment:
`One of two things is necessary...` ; `which is either the PIN code  of the  Skywallet and access to the Skywallet...`